compute_modules/logging/common.py

```diff
@@ -34,17 +34,24 @@
 
 
 # TODO: support for log file output (need access to selected log output location)
-def _create_logger(name: str) -> logging.Logger:
+def _create_logger(
+    name: str,
+    handlers: Optional[Union[logging.Handler, Tuple[logging.Handler, ...]]],
```

Comment:
I have added support for custom handlers, don't know if this is overkill but wanted to make it general (aligned with how sls does it) can scope it to only formatters if needed